modules/javafx.graphics/src/main/java/javafx/stage/Stage.java

```diff
@@ -97,6 +97,7 @@
  * and no decorations.</li>
  * <li>{@link StageStyle#UTILITY} - a stage with a solid white background and
  * minimal platform decorations.</li>
+ * <li>{@link StageStyle#EXTENDED} - a decorated stage with a custom {@link HeaderBar}.</li>
```

Comment:
Do you think it would be helpful to indicate that this is a Preview Feature? Maybe by adding a parenthetical `(Preview Features)` somewhere?